src/Sparrow.Server/Compression/Encoder3Gram.cs

```diff
@@ -496,24 +497,20 @@ private unsafe void BuildDictionary(in FastList<SymbolCode> symbolCodeList)
                     entry.PrefixLength = (byte)symbolLength;
                 }
 
-                if (entry.PrefixLength == 0)
-                {
-                    i--;
-                    continue;
-                }
-                    
-                Debug.Assert(entry.PrefixLength > 0);
-
                 entry.Code = symbolCodeList[i].Code;
 
                 maxBitSequenceLength = Math.Max(maxBitSequenceLength, entry.Code.Length);
                 minBitSequenceLength = Math.Min(minBitSequenceLength, entry.Code.Length);
 
                 tree.Add((uint)entry.Code.Value, entry.Code.Length, (short)i);
+
+                // We update the table.
+                EncodingTable[tableI] = entry;
```

Comment:
We wanted to skip it previously, but now we always assign it. Is this correct?

Reply:
@redknightlois ping